LDAP-Auth/Config/configPage.html

```diff
@@ -139,8 +152,59 @@ <h2 class="sectionTitle">LDAP Settings:</h2>
                     LdapConfigurationPage.txtLdapBindPassword.value = config.LdapBindPassword || "";
                     LdapConfigurationPage.chkEnableUserCreation.checked = config.CreateUsersFromLdap;
                     LdapConfigurationPage.chkEnableCaseInsensitiveUsername.checked = config.EnableCaseInsensitiveUsername;
-                    Dashboard.hideLoadingMsg();
+                    config.EnableAllFolders = config.EnableAllFolders || false;
+                    LdapConfigurationPage.chkEnableAllFolders.checked = config.EnableAllFolders;
+                    /* Default to empty array if Enabled Folders is not set */
+                    config.EnabledFolders = config.EnabledFolders || [];
+                    loadMediaFolders(config).then(() => {
+                      Dashboard.hideLoadingMsg();
+                    });
                 });
+
+                function loadMediaFolders(config) {
+                  if (!LdapConfigurationPage.folderAccessList) {
+                    return Promise.resolve();
+                  }
+
+                  return window.ApiClient.getJSON(window.ApiClient.getUrl('Library/MediaFolders', {
+                    IsHidden: false
+                  })).then((mediaFolders) => {
+                    let html = '';
+                    html += '<h3 class="checkboxListLabel">' + 'Library Access'/*globalize.translate('HeaderLibraries')*/ + '</h3>';
```

Comment:
```suggestion
                    html += '<h3 class="checkboxListLabel">' + 'Library Access' + '</h3>';
```

Reply:
Switched this to using the ${} templating -- I wasn't sure if the code would dynamically translate via HTML injection, but it looks like it does.